tutorials/using-xgboost/src/run.py

```diff
@@ -0,0 +1,82 @@
+# imports
+import mlflow
+import argparse
+import dask_mpi
+
+import xgboost as xgb
+import dask.dataframe as dd
+
+from distributed import Client
+from adlfs import AzureBlobFileSystem
+
+# begin script
+if __name__ == "__main__":
```

Comment:
Suggested pattern is something like this:

```python
import argparse


def main(args):
    # interesting logic here


if __name__ == "__main__":
    parser = argparse.ArgumentParser()
    parser.add_argument("...")

    args = parser.parse_args()
    main(args)
````

This way others can e.g. import `main` from a Jupyter notebook or in unit tests.

Reply:
updated, thanks!